rel/overlay/bin/remsh

```diff
@@ -88,6 +89,9 @@ while getopts ":hn:c:l:mv" optionName; do
     v)
       VERBOSE=0
       ;;
+    t)
+      PATHCONF=$OPTARG
```

Comment:
To match with the `-t ` option perhaps all its TLSCONF. In case we have another PATH variable in the file at some point.

Reply:
changed